docker-compose.yml

```diff
@@ -37,13 +37,16 @@ services:
     image: ${MONTAGU_REGISTRY}/montagu-reverse-proxy:${MONTAGU_PROXY_VERSION}
     ports:
       - "${MONTAGU_PORT}:${MONTAGU_PORT}"
-      - 80:80
+      - "80:80"
+      - "8080:8080"
```

Comment:
I don't think there's any point mapping 8080 here... see proxy PR https://github.com/vimc/montagu-proxy/pull/10#pullrequestreview-83774227

Reply:
I'm a bit confused about how to proceed here - I've exposed this on 8080 just to check that things work.  Once we can restructure the nginx config I agree that this should come out but I'm not sure how to actually do that!

Reply:
Ok yeah I'm with you, maybe you're right that this is the first step.